apps/vendor-portal/src/modules/settings/pages/team/team-list.vue

```diff
@@ -86,7 +86,7 @@
                 $t("SETTINGS.TEAM.PAGES.LIST.TABLE.HEADER.ROLE")
               }}</VcHint>
               <div class="text-ellipsis overflow-hidden whitespace-nowrap mt-1">
-                {{ itemData.item.role }}
+                {{ roleName(itemData.item.role) || "N/A" }}
```

Comment:
"N/A" detection should be placed in the `roleName` method. If you are creating display-formatting logic, it sohuldn't be spread on both the template and script parts.